frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -27,21 +28,47 @@ export enum ClusterUpdateStatus {
 
 export const clusterVersionReference = referenceForModel(ClusterVersionModel);
 
-export const getAvailableClusterUpdates = (cv: ClusterVersionKind): ClusterUpdate[] => {
-  return _.get(cv, 'status.availableUpdates', []);
+const getAvailableClusterUpdates = (cv: ClusterVersionKind): Release[] => {
+  return cv.status?.availableUpdates || [];
 };
 
-export const getSortedUpdates = (cv: ClusterVersionKind): ClusterUpdate[] => {
-  const available = getAvailableClusterUpdates(cv) || [];
+export const getSortedAvailableUpdates = (cv: ClusterVersionKind): Release[] => {
+  const available = getAvailableClusterUpdates(cv);
   try {
     return available.sort(({ version: left }, { version: right }) => semver.rcompare(left, right));
   } catch (e) {
     // eslint-disable-next-line no-console
-    console.error('error sorting cluster updates', e);
+    console.error('error sorting available cluster updates', e);
     return available;
   }
 };
 
+const getConditionalClusterUpdates = (cv: ClusterVersionKind): ConditionalUpdate[] => {
+  return cv.status?.conditionalUpdates || [];
+};
+
+const getNotRecommendedUpdates = (cv: ClusterVersionKind): ConditionalUpdate[] => {
+  return getConditionalClusterUpdates(cv).filter((update) =>
+    update.conditions.find(
+      // Per Trevor King, this condition is subject to change
+      (condition) => condition.type === 'Recommended' && condition.status !== 'True',
```

Comment:
@wking, is the comment on line 53 correct?  If so, we can we expect to know the "final" condition?

Reply:
Risks are polled, so the recommended condition can change over time (maybe the risky state resolved since the last poll, etc.).  Displaying whatever the current state is, with a watch on Cluster version or a poll every few minutes should be fine.

Reply:
The console is already watching the ClusterVersion, so any changes will automatically be rendered in the UI.  It sounds like this check is correct to get any supported but not recommended updates.